src/main/java/net/openhft/chronicle/core/threads/EventLoop.java

```diff
@@ -72,4 +68,12 @@
     static boolean inEventLoop() {
         return CleaningThread.inEventLoop(Thread.currentThread());
     }
+
+    /**
+     * Stop the event loop, then close any resources being held.
+     *
+     * Blocks until the handlers are all stopped and closed
+     */
+    @Override
+    void close();
```

Comment:
Included this here despite it being in Closeable interface (hence the @Override annotation) so we could add that it will block until the loop is closed